docs/topics/Community_Resources.md

```diff
@@ -72,6 +72,13 @@ Using Discord's [Dispatch](#DOCS_DISPATCH_DISPATCH_AND_YOU) tool for game develo
 
 - [JohnyTheCarrot's Dispatch CLI](https://github.com/JohnyTheCarrot/droops-dispatch)
 
+## Website Embed Visualiser
+
+Testing your websites' embeds on Discord is hard. Not only do you have to publish your sites somewhere, which might lead to premature leaking of unannounced projects, but you also have to deal with the caching of embeds Discord does. The following tool aims to mimic Discord's URL embedding system as closely as possible, in real time, so that web devs can test their Discord embeds easily before committing to a website deploy.
```

Comment:
```suggestion
Testing your websites' embeds on Discord is hard. Not only do you have to post your links somewhere to generate an embed, which might lead to premature leaking of unannounced projects, but you also have to deal with the caching Discord does. The following tool aims to mimic Discord's URL embedding system as closely as possible, in real time, so that web devs can test their Discord embeds easily before committing to a website deploy.
```